standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -5026,7 +5026,8 @@ private boolean drop_partition_common(RawStore ms, String catName, String db_nam
         verifyIsWritablePath(partPath);
       }
 
-      if (!ms.dropPartition(catName, db_name, tbl_name, part_vals)) {
+      String partName = Warehouse.makePartName(tbl.getPartitionKeys(), part_vals);
+      if (!ms.dropPartition(catName, db_name, tbl_name, partName)) {
         throw new MetaException("Unable to drop partition");
```

Comment:
Should we throw the exception stack trace/reason for failure?

Reply:
`ms.dropPartition()` will throw the exception, we do not need to throw this `MetaException` here, will remove this code.